sklearn/tests/test_discriminant_analysis.py

```diff
@@ -329,6 +360,60 @@ def test_lda_store_covariance():
     )
 
 
+def test_lda_shrinkage():
+    """
+    Test that shrunk covariance estimator and
+    shrinkage parameter behave the same
+    """
+    X = np.random.rand(100, 10)
```

Comment:
To avoid any potential randomness issue, let's set the RNG at the beginning:

```py
rng = np.random.RandomState(0)
```

and then use it with `rng.randint....`

same for the other test.

If you think it's worth it, we can also parametrize the test with a random seed so that multiple initialization are tested (if the tests are short enough)